testsuite/cluster-test/src/experiments/performance_benchmark.rs

```diff
@@ -61,6 +61,10 @@ pub struct PerformanceBenchmarkParams {
     pub backup: bool,
     #[structopt(long, default_value = "0", help = "Set gas price in tx")]
     pub gas_price: u64,
+    #[structopt(long, help = "Enable periodic stat aggregator.")]
+    pub periodic_stat: bool,
+    #[structopt(long, default_value = "10", help = "Set periodic stat aggregator step")]
+    pub periodic_step: u64,
```

Comment:
Instead of having bool+u64 it would be nicer to have single arg like `periodic_stats: Option<u64>`